app/lib/backend/storage_rewrite.rb

```diff
@@ -1,106 +1,135 @@
-module Backend
-  module StorageRewrite
-
-
-    module_function
-
-    def rewrite_all
-      total_count = Service.count + Cinstance.count + Metric.count + UsageLimit.count
-      progress_each = 100
-      index = 0
+# frozen_string_literal: true
 
-      progress = -> do
-        index += 1
-        break unless (index % progress_each) == 0
-        percent = (index / total_count.to_f) * 100.0
+module Backend
 
-        yield percent if block_given?
-      end
```

Comment:
I'm OK with removing this. But don't you think we'll use this method anymore?

Reply:
I added this feature here: https://github.com/3scale/porta/pull/3595/commits/2a415be08582024e0312777d369c587fe6aa3448

This indeed could be useful for on-premises, with setups that don't have such an excessive amount of stuff, so it can be done without jobs.